server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java

```diff
@@ -381,6 +381,10 @@ protected void handleRetryRollbacking() {
         long now = System.currentTimeMillis();
         for (GlobalSession rollbackingSession : rollbackingSessions) {
             try {
+                //prevent repeated rollback
+                if (rollbackingSession.getStatus().equals(GlobalStatus.Rollbacking) && !rollbackingSession.isTimeout()) {
```

Comment:
if throw RunTimeException it will no longer rollback.

Reply:
I think it will keep rollback